zellij-server/src/panes/plugin_pane.rs

```diff
@@ -254,14 +255,55 @@ impl Pane for PluginPane {
         self.geom.y -= count;
         self.should_render = true;
     }
-    fn scroll_up(&mut self, _count: usize) {
-        //unimplemented!()
-    }
-    fn scroll_down(&mut self, _count: usize) {
-        //unimplemented!()
+    fn scroll_up(&mut self, count: usize) {
+        self.send_plugin_instructions
+            .send(PluginInstruction::Update(
+                Some(self.pid),
+                Event::Mouse(Mouse::ScrollUp(count)),
+            ))
+            .unwrap();
+    }
+    fn scroll_down(&mut self, count: usize) {
+        self.send_plugin_instructions
+            .send(PluginInstruction::Update(
+                Some(self.pid),
+                Event::Mouse(Mouse::ScrollDown(count)),
+            ))
+            .unwrap();
     }
     fn clear_scroll(&mut self) {
```

Comment:
Can you trigger a `clear_scroll` with the mouse? I suppose I assumed it was only really called by things like `reset` in the terminal. If `clear_scroll` isn't something that you can trigger with a mouse, then I'd like to leave this unimplemented for the time-being :)